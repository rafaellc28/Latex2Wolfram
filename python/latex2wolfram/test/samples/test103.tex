% Test determinant matrix
\det \begin{bmatrix}
  6 & 3 & 6\\
  1 & 9 & 5\\
  2 & 5 & 9\\
\end{bmatrix}
