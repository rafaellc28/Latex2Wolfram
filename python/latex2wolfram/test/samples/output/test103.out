det({{6, 3, 6}, {1, 9, 5}, {2, 5, 9}})
